cmd/start_server/start_server.go

```diff
@@ -8,9 +8,25 @@ import (
 	"strconv"
 
 	lsp "github.com/CircleCI-Public/circleci-yaml-language-server/pkg/server"
+	"github.com/rollbar/rollbar-go"
+)
+
+var (
+	RollbarToken = ""
 )
 
 func main() {
+	rollbar.SetToken(RollbarToken)
+	rollbar.SetEnvironment("production")
+	rollbar.SetCodeVersion(lsp.GetServerVersion())
+	rollbar.SetServerRoot("github.com/CircleCI-Public/circleci-yaml-language-server")
+
```

Comment:
I see a couple more things we could configure on Rollbar, here:

* `rollbar.SetServerHost("")`: privacy-wise, I think we shouldn't divulge the user's machine name
* `rollbar.SetPlatform('client')` and use the post_client_item token, [like recommended here for Electron,](https://docs.rollbar.com/docs/electron#node-back-end) that will be better protected against abuse